src/vault/core.clj

```diff
@@ -131,6 +131,44 @@
     "Removes the watch function registered with the given key, if any."))
 
 
+(defprotocol SecretEngine
+  "Basic API for listing, reading, and writing secrets.
+
+  `eng` is a keyword representing the secret engine/mount"
```

Comment:
missed deleting this. if possible reverting one of the old commits would be nicer but if it wasn't factored out cleanly no worries.

Reply:
I moved it back to `Core` and deleted the other file/ns I made for it. I'm not worried about the ballooning issue now

Reply:
Agree, but it was part of a larger commit :/